### PR TITLE
New API eldap:info/1

### PR DESCRIPTION
**Pull request by anupamasingh10**
eldap:info/1 returns information about the LDAP connection (socket and transport protocol)

### CODE REVIEW

File: lib/eldap/doc/src/eldap.xml [hunk 1]
```diff
@@ -548,6 +548,18 @@ Control2 = eldap:paged_result_control(PageSize, Cookie1),
         the series.</p>
       </desc>
     </func>
+    <func>
+      <name since="@OTP-XXXX@">info(Handle) -> connection_info()</name>
```
Comment (IngelaAndin) [inline lib/eldap/doc/src/eldap.xml:552]: You have the ticket number now so you can make it OTP-18480

**Review by KennethL — APPROVED** [review]

**Review by IngelaAndin — APPROVED** [review]

**Review by u3s — COMMENTED** [review]
no comments

File: lib/eldap/src/eldap.erl [hunk 2]
```diff
@@ -608,6 +616,18 @@ loop(Cpid, Data) ->
 	    send(From, Result),
 	    ?MODULE:loop(Cpid, Data);
 
+        {From, info} ->
+            SocketType =
+                case Data#eldap.ldaps of
+                    true ->
```
Comment (vkatsuba) [inline lib/eldap/src/eldap.erl:622]:
```suggestion
                    true ->
```

File: lib/eldap/src/eldap.erl [hunk 3]
```diff
@@ -608,6 +616,18 @@ loop(Cpid, Data) ->
 	    send(From, Result),
 	    ?MODULE:loop(Cpid, Data);
 
+        {From, info} ->
+            SocketType =
+                case Data#eldap.ldaps of
+                    true -> 
+                        ssl;
+                    false ->
```
Comment (vkatsuba) [inline lib/eldap/src/eldap.erl:624]:
```suggestion
                    false ->
```

File: lib/eldap/src/eldap.erl [hunk 4]
```diff
@@ -608,6 +616,18 @@ loop(Cpid, Data) ->
...
 
+        {From, info} ->
+            SocketType =
+                case Data#eldap.ldaps of
+                    true -> 
+                        ssl;
+                    false -> 
+                        tcp
+                end,
+            Res = #{socket => Data#eldap.fd, socket_type => SocketType} ,
```
Comment (vkatsuba) [inline lib/eldap/src/eldap.erl:627]:
```suggestion
            Res = #{socket => Data#eldap.fd, socket_type => SocketType},
```

**Pull request merged**